apps/guide/src/content/03-popular-topics/02-threads.mdx

```diff
@@ -0,0 +1,222 @@
+
+<CH.Code>
+
+```js
+import { ThreadAutoArchiveDuration } from 'discord.js';
+
+const thread = await channel.threads.create({
+	name: 'food-talk',
+	autoArchiveDuration: ThreadAutoArchiveDuration.OneHour,
+	reason: 'Needed a separate thread for food',
+});
+
+console.log(`Created thread: ${thread.name}`);
+```
+
+</CH.Code>
+
+To delete a thread, use the <DocsLink type="class" parent="ThreadChannel" symbol="delete" brackets /> method:
+
+<CH.Code>
+
+```js
+const thread = channel.threads.cache.find((x) => x.name === 'food-talk');
+await thread.delete();
+```
+
+</CH.Code>
+
+## Joining and leaving threads
+
+To join your client to a thread, use the <DocsLink type="class" parent="ThreadChannel" symbol="join" brackets /> method:
+
+<CH.Code>
+
+```js focus=2
+const thread = channel.threads.cache.find((x) => x.name === 'food-talk');
+if (thread.joinable) await thread.join();
+```
+
+</CH.Code>
+
+And to leave one, use <DocsLink type="class" parent="ThreadChannel" symbol="leave" brackets />;
+
+<CH.Code>
+
+```js focus=2
+const thread = channel.threads.cache.find((x) => x.name === 'food-talk');
+await thread.leave();
+```
+
+</CH.Code>
+
+## Archiving, unarchiving, and locking threads
+
+A thread can be either active or archived. Changing a thread from archived to active is referred to as unarchiving the thread. Threads that have _`locked`_ set to _`true`_ can only be unarchived by a member with the _`ManageThreads`_ permission.
+
+Threads are automatically archived after inactivity. "Activity" is defined as sending a message, unarchiving a thread, or changing the auto-archive time.
+
+To archive or unarchive a thread, use the <DocsLink type="class" parent="ThreadChannel" symbol="setArchived" brackets /> method and pass in a boolean parameter:
+
+<CH.Code>
+
+```js focus=2,3
+const thread = channel.threads.cache.find((x) => x.name === 'food-talk');
+await thread.setArchived(true); // Archived.
+await thread.setArchived(false); // Unarchived.
+```
+
+</CH.Code>
+
+This same principle applies to locking and unlocking a thread via the <DocsLink type="class" parent="ThreadChannel" symbol="setLocked" brackets /> method:
+
+<CH.Code>
+
+```js focus=2,3
+const thread = channel.threads.cache.find((x) => x.name === 'food-talk');
+await thread.setLocked(true); // Locked.
+await thread.setLocked(false); // Unlocked.
+```
+
+</CH.Code>
+
+## Public and private threads
+
+Public threads are viewable by everyone who can view the parent channel of the thread. Public threads can be created with the <DocsLink type="class" parent="GuildTextThreadManager" symbol="create" brackets /> method.
+
+<CH.Code>
+
+```js
+import { ThreadAutoArchiveDuration } from 'discord.js';
+
+const thread = await channel.threads.create({
+	name: 'food-talk',
+	autoArchiveDuration: ThreadAutoArchiveDuration.OneHour,
+	reason: 'Needed a separate thread for food',
+});
+
+console.log(`Created thread: ${thread.name}`);
+```
+
+</CH.Code>
```

Comment:
I feel like this part needs to be redone, since we already showed how to create a thread the same way as before. We just need to explain how public/private threads work.